client/utils/items.ts

```diff
@@ -9,6 +11,26 @@ export const validLibraryTypes = [
   'PhotoAlbum'
 ];
 
+export type CardShapes =
+  | 'portrait-card'
+  | 'thumb-card'
+  | 'square-card'
+  | 'banner-card';
+
+/**
+ * Determines if the item is a person
+ *
+ * @param {*} item - The item to be checked.
+ * @returns {boolean} Whether the provided item is of type BaseItemPerson.
+ */
+export function isPerson(item: unknown): item is BaseItemPerson {
```

Comment:
This is already on imageHelper, either remove it from there (preferred as it makes more sense) or remove it from here

Reply:
Removed the other one in 576fdf1.